app/views/warehouse_reports/health/claims/_initial.haml

```diff
@@ -27,10 +27,21 @@
         %th Date Chosen
         %th Qualifying Activities Included
         %th Patients Included
+        %th Result
     %tbody
       - @completed_reports.each do |report|
         %tr
           %td= link_to report.submitted_at.to_date, "#{warehouse_reports_health_claim_url(report)}.txt"
           %td= report.max_date
           %td= report.qualifying_activities.count
-          %td= report.qualifying_activities.select(:patient_id).distinct.count
\ No newline at end of file
+          %td= report.qualifying_activities.select(:patient_id).distinct.count
+          %td
+            - case report.result
+            - when 'accepted'
+              .health_claims__success
```

Comment:
Let's make these link buttons (`ban-success` or `btn-danger`).  They should link to a modal with a summary of what was in the 999, success could, maybe list the number of claims successful?  Failure should list out the failures in a somewhat human readable format, if possible.